library/kong

```diff
@@ -1,11 +1,18 @@
-Maintainers: Mashape Docker Maintainers <support@mashape.com> (@mashape)
-GitRepo: https://github.com/Mashape/docker-kong.git
+Maintainers: Mashape Docker Maintainers <support@konghq.com> (@thekonginc)
+GitRepo: https://github.com/Kong/docker-kong.git
 # needs "Constraints" to match "centos" (which this image is "FROM")
 
-Tags: 0.11-alpine, 0.11.0-alpine
-GitCommit: 55d0b1e894b328483e36b40b178fc080662da773
+Tags: 0.11-alpine, 0.11.1-alpine
+GitCommit: e5bbf3fe757ebe59cfdfda3fade14ef8cd7fa751
 
-Tags: 0.11, 0.11.0, latest
+Tags: 0.11.0-alpine
```

Comment:
Any particular reason to keep this version listed here (and `0.11.0` below)?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
We don't need to keep this tag here, then. Thanks for the catch! I will update this shortly.